### PR TITLE
LOOP-1358: Pump failures

### PR DESCRIPTION
**Pull request by jh-bate**
- basics for checking of pump failures
   -  note could for some as cannot interact with passcode screen

- screens re-org so we can common base screens and functionality
- language clean

### CODE REVIEW

File: src/base/entriesScreen.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    * @param {object} parentScreen.screenTxt
+    * @param {object} parentScreen.generalTxt
+    * @param {object} config
+    */
+    constructor(parentScreen, config) {
+        super(parentScreen);
+        this.config = config;
+    }
+    InfoLabel() {
+        return match.accessible.Label(this.screenTxt.Info);
```
Comment (darinkrauss) [inline src/base/entriesScreen.js]: [nit] `Txt` -> `Text` (not sure if saving one character is worth it)

File: src/homeScreen.js [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+    constructor(language, settingsScreenDefaults) {
+
         this.glucoseScreen = new homeSubScreen.GlucoseScreen(language);
         this.activeInsulinScreen = new homeSubScreen.ActiveInsulinScreen(language);
         this.insulinDeliveryScreen = new homeSubScreen.InsulinDeliveryScreen(language);
         this.activeCarbohydratesScreen = new homeSubScreen.ActiveCarbohydratesScreen(language);
+        this.settingsScreen = new SettingsScreen(language, settingsScreenDefaults);
+        this.bolusScreen = new BolusScreen(language);
+        this.carbEntryScreen = new CarbEntryScreen(language);
+        this.header = new homeSubScreen.Header(language);
```
Comment (darinkrauss) [inline src/homeScreen.js]: [nit] What is the `homeSubScreen`? It is not clear from the variable name.

**Review by darinkrauss — APPROVED** [review]
A couple of nits, but otherwise LGTM! 🔑

File: src/base/entriesScreen.js (hunk 1 above)
Comment (jh-bate) [reply src/base/entriesScreen.js]: lol - yeah

File: src/homeScreen.js (hunk 2 above)
Comment (jh-bate) [reply src/homeScreen.js]: the sub screens that open off the home screen - like the "Active Insulin" section that opens into its own screen when you click it. I could maybe come up with a better name

**Pull request merged**